bids-validator/src/files/nifti.ts

```diff
@@ -0,0 +1,11 @@
+import 'https://raw.githubusercontent.com/rii-mango/NIFTI-Reader-JS/master/release/current/nifti-reader-min.js'
+import { BIDSFileDeno } from './deno.ts'
+
+export function loadHeader(file: BIDSFile) {
+  const header = nifti.readHeader(Deno.readFileSync(file._getPath()).buffer)
```

Comment:
```suggestion
  const buf = file.readBytes(1024)
  const header = globalThis.nifti.readHeader(buf.buffer)
```

Added this in #1536 to keep this from requiring the private path method.